src/routes/userRoutes.js

```diff
@@ -35,5 +35,52 @@ routes.resetPassword = async (req, res) => {
   }
 }
 
+routes.createUser = async (req, res) => {
+  const userInfo = {
+      username: req.body.username,
+      email: req.body.email,
+      password: req.body.password
+  }
+  try {
+      await signUp(userInfo)
+      logger.info('Succeded creating user account', userInfo.username)
+      return res.status(200).json('Succeded creating user account')
+  } catch (err) {
+      logger.error("Creating user failed", userInfo.username, err)
+      return res.status(500).json({error: {message: 'Creating user failed. Please try again'}})
+  }
+}
+
+routes.deleteUser = async (req, res) => {
+  if (!req.params.id) {
+      logger.info("user id was not provided")
+      return res.status(400).json({error: {message: "user id was not provided"}})
+  }
+  const {Accounts} = db.getModels()
+  try {
+      const account = await Accounts.findOne({
```

Comment:
this whole code block doesn't have good indentation. Could you indent please? Like `where` block below should be indented.

```
if (!account) {
  ...everything in here should be indented
}
```